server/cherry_pick.go

```diff
@@ -90,7 +90,8 @@ func (s *Server) handleCherryPick(ctx context.Context, commenter, body string, p
 		return nil
 	}
 
-	args := strings.Split(body, " ")
+	index := strings.Index(body, "/cherry-pick")
```

Comment:
The idea and implementation is very good, but please extract this to a function and write a unit test for it.

Reply:
Ok, will do it today